Makefile

```diff
@@ -96,12 +88,12 @@ build-itest:
 	CGO_ENABLED=0 $(GOBUILD) -tags="rpctest" -o itest/btcd-itest $(BTCD_PKG)
 
 	@$(call print, "Building itest lnd.")
-	CGO_ENABLED=0 $(GOBUILD) -tags="$(ITEST_TAGS)" -o itest/lnd-itest $(ITEST_LDFLAGS) $(LND_PKG)/cmd/lnd
+	CGO_ENABLED=0 $(GOBUILD) -tags="$(ITEST_TAGS)" -o itest/lnd-itest $(DEV_LDFLAGS) $(LND_PKG)/cmd/lnd
 
 install:
 	@$(call print, "Installing tarod and tarocli.")
-	$(GOINSTALL) -tags="${tags}" $(LDFLAGS) $(PKG)/cmd/tarod
-	$(GOINSTALL) -tags="${tags}" $(LDFLAGS) $(PKG)/cmd/tarocli
+	$(GOINSTALL) -tags="${tags}" $(RELEASE_LDFLAGS) $(PKG)/cmd/tarod
```

Comment:
Two things, when I run `make install` I get:
```
$ make install
fatal: No names found, cannot describe anything.
 Installing tarod and tarocli.
GO111MODULE=on go install -v -tags=""  -s -w -buildid= -X github.com/lightninglabs/taro/build.Commit= github.com/lightninglabs/taro/cmd/tarod
flag provided but not defined: -s
usage: go install [build flags] [packages]
Run 'go help install' for details.
make: *** [Makefile:95: install] Error 2
```

And also when I just `make` the `./tarod-debug --version` command doesn't anymore have the commit set.

Reply:
you were right, I forgot the `-ldflags` in front of  `$(RELEASE_LDFLAGS)` same [here](https://github.com/lightningnetwork/lnd/pull/7050)

Thank you for noticing @bhandras :pray: